rest_framework_simplejwt/serializers.py

```diff
@@ -1,11 +1,16 @@
+from django.conf import settings
```

Comment:
Addressed in other comments, but try to use the ROTATE_BLACKLIST something setting rather than checking if the blacklist app is installed. When you change that, make sure you delete this import.